docs/cookbook/modifying-a-template-in-administration.md

```diff
@@ -0,0 +1,133 @@
+# Modifying a Template in Administration
+
+In this tutorial, we'll show you how to proceed if you need to modify a twig template in the administration.
+We will demonstrate this procedure with the example How to display a product transfer status on the product detail page in the administration.
+
+## Adding the information into the template
+
+This tutorial assumes that the product already contains the transfer status attribute.
+If you need to extend the product with a new attribute, see the tutorial [Adding new attribute to an entity](./adding-new-attribute-to-an-entity.md).
+
+Modifying the bundle templates that are located in the vendor can be done via overriding these templates.
+
+*If you need to extend or modify the form itself, it is not necessary to use the overriding and consequently to lose an upgradeability.
+For the extensions of the forms, see [Adding new attribute to an entity](./adding-new-attribute-to-an-entity.md#enable-administrator-to-edit-the-extId-field).*
+
+1. The first step is to create a copy of the original twig template that you want to modify.
+    Because we are trying to override the template located in
+    ```
+    vendor/shopsys/framework/src/Resources/views/Admin/Content/Product/detail.html.twig
+    ```
+    therefore, the new copy must be located in
+    ```
+    app/Resources/ShopsysFrameworkBundle/views/Admin/Content/Product/detail.html.twig
+    ```
+    See in particular at the new directory `ShopsysFrameworkBundle` whose title is based on the name of the bundle with original template.
```

Comment:
I dont think that this sentence makes sense "See in particular at".

Reply:
Fixed